app/src/main/res/drawable/ic_social_facebook.xml

```diff
@@ -1,11 +1,11 @@
 <vector xmlns:android="http://schemas.android.com/apk/res/android"
         android:width="18dp"
-        android:height="16dp"
-        android:viewportWidth="18.0"
-        android:viewportHeight="16.0">
+        android:height="18dp"
+        android:viewportWidth="1024.0"
+        android:viewportHeight="1024.0">
```

Comment:
This was the viewport size in Facebook's original asset. Figured it's alright to keep it around as the `android:height` does the job.